src/fprime_gds/executables/cli.py

```diff
@@ -198,7 +185,6 @@ def parse_args(
         except Exception as exc:
             print(f"[ERROR] {exc}", file=sys.stderr)
             raise
-            sys.exit(-1)
```

Comment:
I removed the optional nature of the ZMQ dependency in the code because we have removed it from the optional dependency of the Python package. It is now a formal dependency in [pyproject.toml](https://github.com/nasa/fprime-gds/blob/devel/pyproject.toml#L36) and part of https://github.com/nasa/fprime/blob/devel/requirements.txt

Reply:
Makes sense!

Reply:
Reverted the call back to sys.exit(), as parse_args() isn't always in a try-catch block across our code base. It's also handled the same way right above.